source/fileio/read_network.py

```diff
@@ -215,7 +217,12 @@ def read(self, flownetwork):
         df_boundary_data.sort_values(self._PARAMETERS["csv_boundary_vs"])
         flownetwork.boundary_vs = df_boundary_data[self._PARAMETERS["csv_boundary_vs"]].to_numpy().astype(np.int)
         flownetwork.boundary_type = df_boundary_data[self._PARAMETERS["csv_boundary_type"]].to_numpy().astype(np.int)
-        flownetwork.boundary_val = df_boundary_data[self._PARAMETERS["csv_boundary_value"]].to_numpy()
+        flownetwork.boundary_val = df_boundary_data[self._PARAMETERS["csv_boundary_value"]].multiply(133.322).to_numpy()  # 133.3223684
+
+        # print(f"Network {self._PARAMETERS['network_name']}")
+        # print("Number of  vs: " + str(flownetwork.nr_of_vs))
+        # print("Number of boundary vs: " + str(len(flownetwork.boundary_vs)))
+        # print("Number of  es: " + str(flownetwork.nr_of_es))
 
```

Comment:
Why do you multiply by 133.322? 
We assume that all imported variables are in SI units. Can you please leave it as it was? 
Also can you please remove the lines 222-225 (I guess they are there just for you, right)?